build.sh

```diff
@@ -9,6 +9,11 @@ INPUT_LINT="${INPUT_LINT:-false}"
 INPUT_DIFF_FILE="${INPUT_DIFF_FILE:-}"
 TEXINPUTS="${TEXINPUTS:-}"
 
+LogAndRun() {
```

Comment:
nit: Would be nice to have an explanation of why this is necessary/helpful here, so it doesn't get refactored out by some poor schmuck like me who didn't look at the blame history.

Reply:
Good point! Probably wrote way more than necessary, but wdyt?

(Sorry for the ugly diff here, I realized I could also remove the unnecessary trampoline that would also cause a poor schmuck refactoring pain)